### ISSUE TITLE
why REGULARIZATION_LOSSES add prelogits_norm？

### ISSUE THREAD

**Issue by ninerui** [davidsandberg/facenet#938]
# Norm for the prelogits

        eps = 1e-5

        prelogits_norm = tf.reduce_mean(tf.norm(tf.abs(prelogits) + eps, ord=args.prelogits_norm_p, axis=1))

        tf.add_to_collection(tf.GraphKeys.REGULARIZATION_LOSSES, prelogits_norm * args.prelogits_norm_loss_factor)

**Comment by ninerui** [comment]
train softmax have add prelogits_norm to regularization loss， why？ prelogits is a output(y)
